shared/locales/phone_types/phone_types.en-US.l20n

```diff
@@ -0,0 +1,25 @@
+/* Field labels */
+<phone_type {
+  *mobile:    "Mobile",
+  home:       "Home",
+  work:       "Work",
+  personal:   "Personal",
+  fax_home:   "Fax home",
+  fax_office: "Fax office",
+  fax_other:  "Fax other",
+  other:      "Other"
+}>
+<_phone_type[$type] {
+  *mobile:    "{{ phone_type.mobile }}",
+  home:      "{{ phone_type.home }}",
+  work:      "{{ phone_type.work }}",
+  personal:  "{{ phone_type.personal }}",
+  faxHome:   "{{ phone_type.fax_home }}",
+  faxOffice: "{{ phone_type.fax_office }}",
+  faxOther:  "{{ phone_type.fax_other }}",
+  other:     "{{ phone_type.other }}"
+}>
```

Comment:
Quite honestly, the more I think about it the more I lean toward simple set of entities:

<phone_type_mobile "Mobile, {{ $carrier }}">
<phone_type_fax "Fax, {{ $carrier }}">

The reason for that is that those are not really *variants* of the same entity (like formFactor stuff). They are separate entities that should be trackable (so that if a localizer does not translate phone_type_fax, we know that it's a missing entity).

If you want to make words like Mobile, Fax etc. reusable, you could do:

<phone_type_mobile "Mobile">
<phone_type_fax "Fax">
<phone_type_and_carrier_mobile "{{ phone_type_mobile }}, {{ $carrier }}">
<phone_type_and_carrier_fax "{{ phone_type_fax }}, {{ $carrier }}">

But I wouldn't use hashes here. Hashes are more for plural variants, or formFactor variants *of the same string*. Those are separate strings I believe.